Mastodon/Scene/Root/MainTab/MainTabBarController.swift

```diff
@@ -292,12 +292,11 @@ extension MainTabBarController {
         tabBarLongPressGestureRecognizer.delegate = self
         tabBar.addGestureRecognizer(tabBarLongPressGestureRecognizer)
 
-        // todo: reconsider the "double tap to change account" feature -> https://github.com/mastodon/mastodon-ios/issues/628
-//        let tabBarDoubleTapGestureRecognizer = UITapGestureRecognizer()
-//        tabBarDoubleTapGestureRecognizer.numberOfTapsRequired = 2
-//        tabBarDoubleTapGestureRecognizer.addTarget(self, action: #selector(MainTabBarController.tabBarDoubleTapGestureRecognizerHandler(_:)))
-//        tabBarDoubleTapGestureRecognizer.delaysTouchesEnded = false
-//        tabBar.addGestureRecognizer(tabBarDoubleTapGestureRecognizer)
+        let tabBarDoubleTapGestureRecognizer = UITapGestureRecognizer()
+        tabBarDoubleTapGestureRecognizer.numberOfTapsRequired = 2
+        tabBarDoubleTapGestureRecognizer.addTarget(self, action: #selector(MainTabBarController.tabBarDoubleTapGestureRecognizerHandler(_:)))
+        tabBarDoubleTapGestureRecognizer.delaysTouchesEnded = false
+        tabBar.addGestureRecognizer(tabBarDoubleTapGestureRecognizer)
```

Comment:
If you just enable it in `MainTabBarcontroller`, this works only for iPhone. Please have a look at `SidebarViewController` as that's the sidbar on iPad :)

Reply:
Oh totally forgot about the iPad support. I will make the necessary change and update you